core/src/main/resources/hudson/slaves/JNLPLauncher/help.properties

```diff
@@ -20,12 +20,15 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-blurb=Allows an agent to be launched using <a href="https://en.wikipedia.org/wiki/Java_Web_Start" target="_blank">Java Web Start</a>.<br>\
-  In this case, a JNLP file must be opened on the agent machine, which will \
-  establish a TCP connection to the Jenkins master.<br>\
-  This means that the agent need not be reachable from the master; the agent \
-  just needs to be able to reach the master. If you have enabled security via \
-  the <i>Configure Global Security</i> page, you can customize the port on \
-  which the Jenkins master will listen for incoming JNLP agent connections.<br>\
-  By default, the JNLP agent will launch a GUI, but it''s also possible to run \
-  a JNLP agent without a GUI, e.g. as a Window service.
+blurb=\
+  Allows an agent to be connected to the Jenkins master whenever it is ready.<br>\
+  In one mode, <a href="https://en.wikipedia.org/wiki/Java_Web_Start" target="_blank">Java Web Start</a> is used. \
+  In this case, a JNLP file must be opened on the agent machine, \
+  which will establish a TCP connection to the Jenkins master. \
+  (Other launch methods use a JNLP file but not Java Web Start, or do not use a JNLP file at all.)<br>\
```

Comment:
For the last clause, see https://github.com/jenkinsci/docker-jnlp-slave/issues/87.